R/connection-pane.R

```diff
@@ -2,6 +2,11 @@
 # - can only return so many objects at once, how to paginate via UI?
 #   - impacts experiments, model registry, etc.
 
+# adding new environment to hold the data.
+brixter.env <- new.env()
```

Comment:
Added **brixter.env** to hold the data which can then be filtered without API calls